### DISCUSSION TITLE
CodeCov is deprecated

### DISCUSSION THREAD

**Discussion by oscarmcm** [django-wiki/django-wiki#1279] [Q&A]
Seems that CodeCov is yanked from PyPI https://pypi.org/project/codecov/

There's a discussion happening here:

https://community.codecov.com/t/codecov-yanked-from-pypi-all-versions/4259

So far, this doesn't impact the CodeCov GitHub action but we might need to have all eyes open to find other alternatives as this affects fresh installs of the `default` and `test` environments, meaning that `circleci` future runs will fail at this moment.

https://github.com/django-wiki/django-wiki/blob/main/pyproject.toml#L89

**Comment by oscarmcm** [comment]
Tagging @benjaoming here

**Reply by benjaoming** [reply]
Maybe we can use the `codecov/codecov` orb for Circle CI?

**Reply by oscarmcm** [reply]
There's a migration guide https://docs.codecov.com/docs/deprecated-uploader-migration-guide#python-uploader

**Reply by oscarmcm** [reply]
And seems that Pillow has already a solution https://github.com/python-pillow/Pillow/pull/7085/files

**Reply by oscarmcm** [reply]
Here's the new uploader https://github.com/codecov/uploader

**Reply by oscarmcm** [reply]
> Maybe we can use the `codecov/codecov` orb for Circle CI?

Might work for Circle CI but will fail for local environments...

So the question here is that if it's ok to have codecov fully disabled on local development?

**Reply by benjaoming** [reply]
With CodeCov's security history in mind, I think it's better to use a versioned Orb than a curl command :)

https://circleci.com/developer/orbs/orb/codecov/codecov

**Reply by benjaoming** [reply]
Does it make sense to run CodeCov outside CI?

**Reply by oscarmcm** [reply]
From my point of view, it doesn't make sense

**Reply by benjaoming** [reply]
Same for me, I think it's fine to choose the Orb for now :+1:
